pro/booking/classes/consignment-request/class-bring-booking-consignment-request.php

```diff
@@ -113,7 +113,9 @@ public function get_sender_address() {
 
 		$sender = $this->get_sender();
 
-		return [
+		return apply_filters(
+      'bring_fraktguiden_get_consignment_sender_address',
```

Comment:
The indentation is wrong. Please use tabs.

Reply:
Should be OK now.